text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,675 @@
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC codifies the requirements for each target tier, and for moving targets
+to a different tier.
+
+# Motivation
+[motivation]: #motivation
+
+Rust developers regularly implement new targets in the Rust compiler, and
+reviewers of Rust pull requests for such new targets would like a clear,
+consistent policy to cite for accepting or rejecting such targets. Currently,
+individual reviewers do not know what overall policy to apply, and whether to
+apply solely their own judgment or defer to a Rust governance team.
+
+Rust developers regularly ask how they can raise an existing target to tier 2
+(and in particular how they can make it available via `rustup`), and
+occasionally ask what it would take to add a new tier 1 target. The Rust
+project has no clear policy for target tiers. People not only don't know, they
+don't know who to ask or where to start.
+
+See <https://forge.rust-lang.org/release/platform-support.html> for more
+information about targets and tiers.
+
+Once accepted, the policy sections of this RFC should be posted on
+<https://forge.rust-lang.org/> in a "Target Tier Policy" section; this RFC will
+not be the canonical home of the up-to-date target tier policy.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Rust provides three tiers of target support:
+
+- Rust provides no guarantees about tier 3 targets; they exist in the codebase,
+  but may or may not build.
+- Rust's continuous integration checks that tier 2 targets will always build,
+  but they may not pass tests.
+- Rust's continuous integration checks that tier 1 targets will always build
+  and pass tests.
+
+Adding a new tier 3 target imposes minimal requirements; we focus primarily on
+avoiding disruption to other ongoing Rust development.
+
+Tier 2 and tier 1 targets place work on the Rust community as a whole, to avoid
+breaking the target. Thus, these tiers require commensurate and ongoing efforts
+from the maintainers of the target, to demonstrate value and to minimize any
+disruptions to ongoing Rust development.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Rust provides three tiers of target support:
+
+- Rust provides no guarantees about tier 3 targets.
+- Rust's continuous integration checks that tier 2 targets will always build,
+  but they may or may not pass tests.
+- Rust's continuous integration checks that tier 1 targets will always build
+  and pass tests.
+
+This policy defines the requirements for accepting a proposed target at a given
+level of support.
+
+Each tier builds on all the requirements from the previous tier, unless
+overridden by a stronger requirement. Tier 2 and tier 1 also provide additional
+requirements to be met if supplying host tools for the target; a target at tier
```

Comment:
This is the first mention of "host tools", which are only incidentally defined in the middle of a future paragraph. Might be nice to define on first usage.

Reply:
Good catch. Done.